dotcom-rendering/src/web/components/SupportingContent.tsx

```diff
@@ -0,0 +1,100 @@
+import { css } from '@emotion/react';
+import { ArticleDesign } from '@guardian/libs';
+import { from, until } from '@guardian/source-foundations';
+import { CardHeadline } from './CardHeadline';
+
+type Props = {
+	supportingContent: DCRSupportingContent[];
+	imagePosition: ImagePositionType;
+};
+
+const wrapperStyles = css`
+	position: relative;
+	display: flex;
+	margin-left: 5px;
+	margin-right: 5px;
+`;
+
+const directionStyles = (imagePosition: ImagePositionType) => {
+	switch (imagePosition) {
+		case 'left':
+		case 'right':
+			return css`
+				flex-direction: column;
+				${from.phablet} {
+					flex-direction: row;
+				}
```

Comment:
Looking at Frontend, this should have switched from the `tablet` breakpoint.

Fixed in [ec72534](https://github.com/guardian/dotcom-rendering/pull/6598/commits/ec72534d72b91c11a6a838e96ed0cb152a81d594)